shared/js/otSdkDriver.js

```diff
@@ -776,6 +789,21 @@ loop.OTSdkDriver = (function() {
     },
 
     /**
+     * Sends the cursor position on the data channel.
+     *
+     * @param {String} message The message to send.
+     */
+    sendCursorMessage: function(message) {
+      if (!this._publisherChannel || !this._subscriberChannel) {
+        return;
```

Comment:
Also, I'm not sure if you're going to address it here or not, but we need to make sure that we create a separate data channel for the pointer sharing.

This means we won't flood the text chat channel, and we can also likely make the pointer data channel lossy (we'll need to investigate if we're able to supply parameters to it and what they should be).